### PR TITLE
Split Up Instance scripts

### PR DESCRIPTION
**Pull request by aaron02**
**Description**
Split up Instance Scripts to make them Better readable and Clear. For now only Splitted up the Rewritten or new ones.

**Todo / Checklist**
- [x] Target is branch develop.
- [x] Detailed description about this pull request.
- [x] Checked AE-Coding standards.

**Tests Performed:** 
- [x] Build AE.
- [x] Server startup.
- [x] Log into world.

### CODE REVIEW

File: src/scripts/InstanceScripts/CMakeLists.txt [hunk 1]
```diff
@@ -25,8 +97,80 @@ set(SRC_BASIC
 set(sources
     ${SRC_BASIC}
     ${SRC_CLASSIC_SCRIPTS_FILES}
+	${SRC_CLASSIC_BLACKFATHOM_SCRIPTS_FILES}
```
Comment (schnek) [inline src/scripts/InstanceScripts/CMakeLists.txt:100]: tab

File: src/scripts/InstanceScripts/Wotlk/IcecrownCitadel/LadyDeathwhisper.hpp [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    CreatureAISpells* darkMartyrdomSpell;
+};
+
+//////////////////////////////////////////////////////////////////////////////////////////
+/// Spell: Cultist Dark Martyrdom
+class DarkMartyrdom : public SpellScript
+{
+public:
+    void afterSpellEffect(Spell* spell, uint8_t effIndex) override;
+};
```
Comment (Appled) [inline src/scripts/InstanceScripts/Wotlk/IcecrownCitadel/LadyDeathwhisper.hpp:118]: Missing newline

File: src/scripts/InstanceScripts/Wotlk/IcecrownCitadel/LordMarrowgar.hpp [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+class BoneSlice : public SpellScript
+{
+public:
+    SpellScriptEffectDamage doCalculateEffect(Spell* spell, uint8_t effIndex, int32_t* dmg) override;
+    SpellCastResult onCanCast(Spell* spell, uint32_t* /*parameter1*/, uint32_t* /*parameter2*/) override;
+    void filterEffectTargets(Spell* /*spell*/, uint8_t effectIndex, std::vector<uint64_t>* effectTargets) override;
+    void afterSpellEffect(Spell* spell, uint8_t effIndex) override;
+
+    uint32_t targetCount;
+};
```
Comment (Appled) [inline src/scripts/InstanceScripts/Wotlk/IcecrownCitadel/LordMarrowgar.hpp:164]: Missing newline

File: src/world/Objects/Units/Creatures/AIInterface.cpp [hunk 4]
```diff
@@ -1068,6 +1068,10 @@ void AIInterface::updateTargets(unsigned long time_passed)
         // If unit has target but does not have threat with it, evade
         if (getUnit()->getThreatManager().canHaveThreatList() && !getUnit()->getThreatManager().isThreatenedBy(getCurrentTarget()))
         {
+            auto test = getCurrentTarget();
+            bool threat = getUnit()->getThreatManager().isThreatenedBy(test);
+            auto list = getUnit()->getThreatManager().getModifiableThreatList();
```
Comment (Appled) [inline src/world/Objects/Units/Creatures/AIInterface.cpp:1073]: Are these still needed?

**Review by Appled — APPROVED** [review]

**Pull request merged**